js/projections.js

```diff
@@ -20,9 +20,7 @@
 		factory(Highcharts);
 	}
 }(function (H) {
-
-	var PlaneProjection,
-		LineProjection,
+	const perspective = H._modules['Core/Math3D.js'].perspective,
```

Comment:
One more thing, sorry! I suggested `const`, but we're still using ES5 here so `var` would be better